CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+### 1.2.0
```

Comment:
For version 1.5x OpenCart there is no abandoned cart functionality so the latest update was version 1.1.4

Reply:
Removed from changelog.